hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -323,7 +333,7 @@ async def _format_trading_rules(self, instruments_info: Dict[str, Any]) -> Dict[
         trading_rules = {}
         for rule in instruments_info["data"]:
             try:
-                trading_pair = await OBDS.trading_pair_associated_to_exchange_symbol(rule["symbol"])
+                trading_pair = await AscendExAPIOrderBookDataSource.trading_pair_associated_to_exchange_symbol(rule["symbol"])
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols